# ChIP-seq
My collection of resources for ChIP-seq analysis. I needed a place to compile and organise all my notes - so here it is!


### Key papers


#### About ChIP-seq in general:

* [ChIPâ€“seq: advantages and challenges of a maturing technology](https://www.nature.com/nrg/journal/v10/n10/full/nrg2641.html)

* [Role of ChIP-seq in the discovery of transcription factor binding sites, differential gene regulation mechanism, epigenetic marks and beyond](http://www.tandfonline.com/doi/full/10.4161/15384101.2014.949201) (This is a good general introduction)

* [ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia](http://genome.cshlp.org/content/22/9/1813.long)

#### About ChIP-seq analysis specifically:

* [Practical Guidelines for the Comprehensive Analysis of ChIP-seq Data](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003326)

* [Recent advances in ChIP-seq analysis: from quality management to whole-genome annotation](https://academic.oup.com/bib/article-lookup/doi/10.1093/bib/bbw023) 

* [ChIP-Seq: technical considerations for obtaining high-quality data](https://www.nature.com/ni/journal/v12/n10/full/ni.2117.html)

* [Analytical tools and current challenges in the modern era of neuroepigenomics](http://www.nature.com/neuro/journal/v17/n11/full/nn.3816.html) (relevant outside neuroepigenomics...)

### Useful presentations

* [Whitehead Institute AnalysisofChIP-seqData2017](http://jura.wi.mit.edu/bio/education/hot_topics/ChIPseq_2017/AnalysisofChIP-seqData2017.pdf)

* [bioinformatics-analysis-of-chip-seq-data: Phil Ewels](http://tallphil.co.uk/bioinformatics-analysis-of-chip-seq-data/)


### Workflows

* [NGI-ChIPseq: SciLife Lab](https://github.com/SciLifeLab/NGI-ChIPseq)

### Visualisation

* [Biostars: Visualizations of ChIP-Seq data using Heatmaps](https://www.biostars.org/p/180314)
